windows_containers/wmco_rn/windows-containers-release-notes-10-18-x.adoc

```diff
@@ -0,0 +1,49 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="windows-containers-release-notes-10-18-x"]
+= {productwinc} release notes
+include::_attributes/common-attributes.adoc[]
+:context: windows-containers-release-notes
+
+toc::[]
+
+
+[id="wmco-10-18-0"]
+== Release notes for Red Hat Windows Machine Config Operator 10.18.0
+
+Issued: 19 March 2025
+
+This release of the Windows Machine Config Operator (WMCO) provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 10.18.0 were released in link:https://access.redhat.com/errata/RHBA-2025:3040[RHBA-2025:3040].
+
+[id="wmco-10-18-0-new-features"]
+=== New features and improvements
+
+[id="wmco-10-18-0-new-features-hpa"]
+==== Horizontal Pod autoscaling is available for Windows workloads
+You can now use a horizontal pod autoscaler (HPA) to scale your Windows pods based on CPU and/or memory resource metrics. For more information on using an HPA, see xref:../../nodes/pods/nodes-pods-autoscaling.adoc#nodes-pods-autoscaling[Automatically scaling pods with the horizontal pod autoscaler].
+
+[id="wmco-10-18-0-new-features-eus"]
+==== Control Plane Only updates are now available for Windows nodes
+You can now use *Control Plane Only* updates, previously known as *EUS-to-EUS* updates, to update your Windows nodes between {product-title} EUS versions. For more information, see xref:../../windows_containers/windows-node-upgrades.adoc#wmco-upgrades-eus_windows-node-upgrades[Windows Machine Config Operator Control Plane Only upgrade].
+
+[id="wmco-10-18-0-new-features-metrics"]
+==== WMCO metrics endpoint is now HTTPS
+The WMCO metrics endpoint is now exposed over HTTPS. Previously, WMCO pod metrics were available over HTTP. This change ensures that the WMCO metrics endpoint is secure.
```

Comment:
I worry sometimes about strong absolute statements, esp wrt security. I don't know enough about this, but would it make sense to say:
```suggestion
The WMCO metrics endpoint is now exposed over HTTPS. Previously, WMCO pod metrics were available over HTTP. This change improves the security posture of the WMCO metrics endpoint.
```